src/docs/implementation/QuickStart_arcCommander_expert.md

```diff
@@ -38,6 +38,8 @@ arc a add -s <YourStudyID> -a <YourAssayID>
 arc sync -f -r https://git.nfdi4plants.org/<YourUserName>/<YourARC> -m "initialize ARC structure"
 ```
 
+> 💡 The last command will prompt an Error `ERROR: GIT: fatal: repository 'https://git.nfdi4plants.org/<YourUserName>/<YourARC>' not found`. Don't worry, the ARC is being created at that address. 
```

Comment:
ERROR: GIT: fatal: couldn't find remote ref main is shown in my case (macOS)

Reply:
Happens when the arc is not prepared on the hub before, so it could even be quicker than the desciption above by scipping 1.&2.
It might be nice to point out that the arc is also in this method private by default.